starknet_py/net/account/account_client.py

```diff
@@ -309,7 +308,7 @@ async def _get_max_fee(
     ) -> int:
         if auto_estimate and max_fee is not None:
             raise ValueError(
-                "Max_fee and auto_estimate are exclusive and cannot be provided at the same time."
+                "Parameters max_fee and auto_estimate are mutually exclusive."
```

Comment:
Should we mix words `parameter` and `argument` or stick to a single one in every exception?

cc @Solpatium @war-in